modules/client-configuration/pages/contact-methods-traditional.adoc

```diff
@@ -3,14 +3,46 @@
 
 Traditional clients can communicate with the {productname} Server using a range of methods.
 
-The {productname} daemon ([command]``rhnsd``) runs on traditional client systems and periodically connects with {productname} to check for new updates and notifications.
+The lightweight {productname} daemon ([command]``rhnsd``) runs on traditional client systems.
+The daemon periodically connects with {productname} to check for new updates and notifications.
+
+[WARNING]
+====
+The [command]``rhnsd`` method has serious limitations.
+It does not scale as well as Salt.
+Newer operating systems are not supported and will not be added in the future.
+====
+
+For more information, see xref:client-configuration:contact-methods-rhnsd.adoc[].
+
+Traditional with OSAD::
+is the same as traditional but allows the server to
+push updates to clients.
+OSAD is an enhancement to [command]``rhnsd``
+OSAD allows traditional clients to execute scheduled actions immediately.
+It does not apply to Salt clients.
+////
+Requires rhnsd and another additional daemon
+(osad) on each client. Same restrictions apply
+////
+
+
+Traditional SSH Push::
+is same as traditional but allows the server to
+push updates to clients, using the SSH protocol as a
+transport layer.
+////
+Requires rhnsd and sshd. Optionally allows to also tunnel
+HTTP/HTTPS traffic (for package download) via SSH: this is called SSH
+Push with Tunnel
+////
 
 Push via SSH is used in environments where clients cannot reach the {productname} Server directly.
 In this environment, clients are located in a firewall-protected zone called a DMZ.
 No system within the DMZ is authorized to open a connection to the internal network, including the {productname} Server.
 
-OSAD is an alternative contact method between {productname} and traditional clients.
-OSAD allows traditional clients to execute scheduled actions immediately.
+Traditional SSH Push with Tunnel::
+The same as SSH Push, but tunnels HTTP/HTTPS traffic (for package download) via SSH.
```

Comment:
This message is also true for salt. Not sure if we have this somewhere.

Reply:
ok, also tracked separately now.